CollAction/Services/ProjectService.cs

```diff
@@ -77,7 +77,7 @@ public ProjectService(ApplicationDbContext context, IStringLocalizer<IProjectSer
                     Target = p.Target,
                     Participants = p.Participants.Count()
                 })
-               .ToListAsync();
+               .ToList();
```

Comment:
Is this on purpose? Making it synchronous ensures a thread is occupied while it queries the database.